hdrreport/hdrreport.go

```diff
@@ -37,32 +48,16 @@ func WriteReportCSV(filename *string, hist *hdrhistogram.Histogram) error {
 }
 
 func PrintLatencySummary(hist *hdrhistogram.Histogram) {
-	fmt.Printf("FROM    TO #REQUESTS\n")
-	fmt.Printf("   0     2 %d\n", SumBars(0, 2, hist.Distribution()))
-	fmt.Printf("   2     8 %d\n", SumBars(2, 8, hist.Distribution()))
-	fmt.Printf("   8    32 %d\n", SumBars(8, 32, hist.Distribution()))
-	fmt.Printf("  32    64 %d\n", SumBars(32, 64, hist.Distribution()))
-	fmt.Printf("  64   128 %d\n", SumBars(64, 128, hist.Distribution()))
-	fmt.Printf(" 128   256 %d\n", SumBars(128, 256, hist.Distribution()))
-	fmt.Printf(" 256   512 %d\n", SumBars(256, 512, hist.Distribution()))
-	fmt.Printf(" 512  1024 %d\n", SumBars(512, 1024, hist.Distribution()))
-	fmt.Printf("1024  4096 %d\n", SumBars(1024, 4096, hist.Distribution()))
-	fmt.Printf("4096 16384 %d\n", SumBars(4096, 16384, hist.Distribution()))
-}
```

Comment:
Just curious, what's the motivation for replacing this latency summary?

Reply:
Ohhhh, nevermind me -- I see now that the old summary only worked for milliseconds.